app/assets/javascripts/avalon/media-player.es6

```diff
@@ -105,11 +101,15 @@ export default class MediaPlayer {
     
     
     playFromHash() {
+        if (this.timer) {
+            clearTimeout(this.timer)
+            this.timer = null
+        }
         var mediaFragment = this.getMediaFragment(window.location.hash)
         var mediaPlayer = document.getElementById('iiif-av-player')
         mediaPlayer.setCurrentTime(mediaFragment.start)
         mediaPlayer.play()
-        setTimeout(() => {
+        this.timer = setTimeout(() => {
```

Comment:
I don't think we should automatically pause the player. It makes for a weird user experience. In current Avalon, we just let the player continue playing, and update the various UI elements that indicate which structural element is currently playing.